prisma/seed.ts

```diff
@@ -26,9 +27,29 @@ async function seed() {
         firstName,
         lastName,
         email,
+        groups,
+      },
+      {
+        username: 'user1',
+        firstName: 'Johannes',
+        lastName: 'Kepler',
+        email: 'user1@myseneca.ca',
+        groups: 'mycustomdomain-faculty',
+      },
+      {
+        username: 'user2',
+        firstName: 'Galileo',
+        lastName: 'Galilei',
+        email: 'user2@myseneca.ca',
+        groups: 'mycustomdomain-students',
+      },
+      {
+        username: 'user3',
+        firstName: 'Adam',
+        lastName: 'Mason',
+        email: 'user3@myseneca.ca',
+        groups: 'mycustomdomain-faculty, mycustomdomain-admins',
```

Comment:
It will only ever be a single string

Reply:
Are you saying an user will only have one group?

Reply:
This is also what I understood from our conversations yesterday. I think users will only ever have 1 group they belong to.